### PR TITLE
feat(vm): default vm handler

### PR DESCRIPTION
**Pull request by saitofun**

### CODE REVIEW

File: vm/vm.go [hunk 1]
```diff
@@ -40,3 +41,16 @@ func NewHandler(endpoints map[Type]string) *Handler {
...
+func init() {
+	var endpoints = make(map[Type]string)
+	for key, typ := range vmEndpointConfigEnvKeyMap {
+		if ep := viper.GetString(key); ep != "" {
+			endpoints[typ] = ep
+		}
+	}
+
+	DefaultHandler = NewHandler(endpoints)
+}
```
Comment (CoderZhi) [inline vm/vm.go:56]: why we need it be initialized in `init`?

Comment (saitofun) [reply vm/vm.go:56]: I tried to initialize a default vm handler, if the vm endpoint can be read from env. And then the `vm.DefaultHandler` will be used to initialize the `msg.DefaultHandler`.

Comment (saitofun) [reply vm/vm.go:56]: if some module don't need the default handler, they can create a hander by `NewHander`

Comment (huangzhiran) [reply vm/vm.go:56]: i  think default if unnecessary， just NewHandler is more clear

**Pull request closed without merging**